IdentityServer/v7/docs/content/diagnostics/otel/metrics.md

```diff
@@ -7,66 +7,74 @@ weight: 10
 (added in v7.0)
 
 The metric counters are designed to provide a high level overview. They are designed to not contain any
-sensitive information. The counters may contain tags to indicate the source of the data.
+sensitive information. The counters may contain tags to indicate the source of the data. Metrics are
+intended to be used to provide overall health metrics and are typically used to show graphs on
+a dashboard or to set up monitoring rules. If the metrics indicates that something has happened,
+the traces and logs are used to find out the reason. Open Telemetry monitoring tools might have
+provide features to find the traces and logs corresponding to certain metrics.
+
+IdentityServer emits metrics from the IdentityServer library. Our quick start for the UI also
+[contains metrics](#metrics-in-the-ui) that can be used as a starting point for monitoring UI events.
```

Comment:
I would move the "no sensitive information" and "may contain flags" to the end of this paragraph:

The metric counters that IdentityServer emits are designed to not contain any sensitive information. They often are tagged to indicate the source of the events.